### PR TITLE
add: feature/se-57/fix

### PR DESCRIPTION
**Pull request by 11Meru11**
closes SE2-WE23/documentation#57

Page to set the neccessary skills for the job

### CODE REVIEW

File: src/routes.jsx [hunk 1]
```diff
@@ -48,7 +48,15 @@ const ROUTES = [
...
-    element: <EmployeeOverviewPage />
+    element: <EmployeeOverviewPage />,
+  },
+  {
+    path: "job-description",
+    element: <JobDescriptionPage />,
+  },
+  {
+    path: "job-description",
+    element: <JobDescriptionPage />,
```
Comment (Arkcorner) [inline src/routes.jsx]:
```suggestion
  {
    path: "job-description",
    element: <JobDescriptionPage />,
```
Job description is doubled in route

File: src/routes.jsx
Comment (Arkcorner) [inline src/routes.jsx:1]: please add the import for your page at the top

File: src/pages/JobDescription/JobDescriptionPage.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          display: "flex",
+          justifyContent: "center",
+          marginTop: "50px",
+          marginBottom: "50px",
+        }}
+      >
+        {mockedDescription?.map((jobDescription) => (
+          <JobDescriptionItem jobDescription={jobDescription} key={uuidv4()} />
+        ))}
+      </Grid>
```
Comment (Arkcorner) [inline src/pages/JobDescription/JobDescriptionPage.jsx:27]: Please adjust spacing to be more like the figma (same distance in between the items at the sides and top/bottom)

File: src/pages/JobDescription/JobDescriptionPage.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { Button, Grid } from "@mui/material/";
+import { v4 as uuidv4 } from "uuid";
+import PageLayout from "../../components/PageLayout";
+import JobDescriptionItem from "./components/JobDescribtionItem";
+import mockedDescription from "./mockData";
+
+function JobDescriptionPage() {
+  return (
+    <PageLayout title="Job Description">
+      <h2>Select relevant Skills for the job</h2>
```
Comment (Arkcorner) [inline src/pages/JobDescription/JobDescriptionPage.jsx:10]: please add positioning so that its not completely stuck in the top left corner

**Review by Arkcorner — CHANGES_REQUESTED** [review]

**Comment by julix14** [comment]
Wrong branch naming.

**Pull request closed without merging**